Dockerfile

```diff
@@ -23,7 +23,13 @@
 # Dockerfile for cray-console-node service
 
 # Build will be where we build the go binary
-FROM arti.dev.cray.com/baseos-docker-master-local/sles15sp2:sles15sp2 as build
+FROM arti.dev.cray.com/baseos-docker-master-local/sles15sp3:sles15sp3 as build
```

Comment:
any reason we aren't using artifactory.algol60.net/registry.suse.com/suse/sle15:15.3 ?

Reply:
Yeah, we tried that, but it doesn't have all of the package sources needed. Specifically, it doesn't have a package source for go. After discussing with @mtupitsyn , he recommended that we stay with the arti image.

Reply:
See [this Slack thread](https://cray.slack.com/archives/G01GNSCG6KV/p1643301307072100?thread_ts=1643236382.051700&cid=G01GNSCG6KV) for full context.

Reply:
I'll pull you into the Slack thread where @mtupitsyn and I discussed this, so you can get full context.

Reply:
Discussed this offline with Randy., He is going to pursue this issue separately, but is content with this merging in the meantime.